Lib/distutils/version.py

```diff
@@ -299,9 +304,9 @@ class LooseVersion (Version):
 
     component_re = re.compile(r'(\d+ | [a-z]+ | \.)', re.VERBOSE)
 
-    def __init__ (self, vstring=None):
-        if vstring:
-            self.parse(vstring)
+    def __init__(self, vstring=None):
```

Comment:
Is this needed?
I think you could not define `__init__` and let the parent class do the initialization

Reply:
`self.vstring` is only used by `LooseVersion`, and not needed in `Version` or `StrictVersion`, which is why I left the `__init__` here, for the purpose of initializing `self.vstring`.